cmd/tier-journal.go

```diff
@@ -56,23 +56,29 @@ var (
 	errUnsupportedJournalVersion = errors.New("unsupported pending deletes journal version")
 )
 
-func initTierDeletionJournal(done <-chan struct{}) (*tierJournal, error) {
-	diskPath := globalEndpoints.FirstLocalDiskPath()
-	j := &tierJournal{
-		diskPath: diskPath,
-	}
+func initTierDeletionJournal(ctx context.Context) (*tierJournal, error) {
+	diskPaths := globalEndpoints.LocalDisksPaths()
+	for _, diskPath := range diskPaths {
+		j := &tierJournal{
+			diskPath: diskPath,
+		}
```

Comment:
the tier deletion journal is intentionally being done centrally as it an overkill to have it run on all disk paths. @krisis PTAL

Reply:
@vadmeste to select the first available local disk on a server works to an extent. It's definitely an improvement. Additionally, we need to act on journals that may be present on other local disks, due to a different set of disks available locally during previous server restarts.

E.g 
1. On the first startup, when all disks are online, this approach would select the first local disk on each server.
2. Imagine the first disk on the first server goes temporarily offline. On restart, we would select the second disk on this server.
3. Now the first disk is back online. On a subsequent restart, we would pick this disk. 

At this point we would have a journal in the second disk which will remain unattended. We need to address this too.

Reply:
@vadmeste we could address the issue of possibly unattended tier deletion journals on other local disks in a separate PR.

Reply:
Yes, but I just thought this cannot be more bad than what we have right now. Yes, we can revert this and address it more properly in another PR.